addons/product/models/product_attribute.py

```diff
@@ -248,8 +256,10 @@ class ProductTemplateAttributeValue(models.Model):
 
     @api.multi
     def name_get(self):
-        if not self._context.get('show_attribute', True):  # TDE FIXME: not used
-            return super(ProductTemplateAttributeValue, self).name_get()
+        """Override because in general the name of the value is confusing if it
```

Comment:
can we keep old context key here to allow to keep old behavior.
And add key where we want the new nameget?